### PR TITLE
Add explanation for each section

### LINKED ISSUE #156: Fix quick start typo
"Operation system" should be "Operating system".

### LINKED ISSUE #161: Explanation for each section in UG

### PR DESCRIPTION
**Pull request by eeliana**
fix #161 
fix #156

### CODE REVIEW

**Comment by eeliana** [comment]
> Wait why did codecov increase HAHAHA

I don't know eh I only changed one file XD

File: docs/UserGuide.md [hunk 1]
```diff
@@ -256,6 +286,29 @@ Examples:
...
+A `Lesson` _**must**_ have the following fields: a start `date`, a `time range` and a `subject`.
+
+A `Lesson` can be categorised into 2 types: 
+1. A _**weekly**_ recurring lesson
+2. A one-off makeup lesson.
+
+The essential fields for a lesson are:
+* `date`
+* `time range`
+* `subject`
```
Comment (angkoonhwee) [inline docs/UserGuide.md:299]:
```suggestion
* `subject`
*`lesson rates`
```

Comment (angkoonhwee) [reply docs/UserGuide.md:299]: Can note somewhere that the `lesson rates` refer to cost per hour as well

Comment (angkoonhwee) [reply docs/UserGuide.md:299]: can also say that this affects the automated fees calculation

File: docs/UserGuide.md [hunk 2]
```diff
@@ -81,10 +82,14 @@ to help you with the installation. Follow the guide for your operation system fo
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+  
+* There are constraints in place to determine whether the value you provide for a field is valid.
```
Comment (eeliana) [inline docs/UserGuide.md:86]:
```suggestion
* There are constraints in place to determine whether the value you provided for a field is valid. 
```

**Pull request merged**